mamba/mamba/shell_templates/micromamba.nu

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env nu
+def-env "micromamba activate"  [name: string] {
+  # get envs and update current env to path
+  let envs = (
+     (micromamba env list --json | from json).envs | reduce -f {} {|it, acc|
```

Comment:
If this is used for Mamba, you can't use `micromamba` here

Reply:
ah I will undo the mamba for now, and do a separate branch for mamba.